app/views/feedback/new.html.haml

```diff
@@ -0,0 +1,6 @@
+%section.form_wrapper
+  = simple_form_for(@form, url: feedback_path) do |f|
+    = f.hidden_field :url
+    = f.input :email, as: :email
+    = f.input :comment, as: :text
+    = f.button :submit, 'Send Feedback', role: 'button'
```

Comment:
Oh, and on the topic of `accessibility_spec.rb`, if you add the feedback pages to `config/sitemap.rb` (in the public group because they are) then the accessibility test that runs through a sitemap will also check them.